packages/subscribe-to-stanford/manifest.json

```
@@ -0,0 +1,88 @@
+{
+  "$schema": "http://localhost:3000/schemas/v1.json#",
+  "name": "subscribe-to-stanford-report",
+  "version": "1.0.0",
```

Set version to 4.0.1 
```suggestion
  "version": "4.0.1",
```

---

Ok, changed + I'll try to remember about that for the future components!